### DISCUSSION TITLE
Ultima: Template layout and Angular 18

### DISCUSSION THREAD

**Discussion by kashcode** [primefaces/community#2120] [PrimeNG Templates]
Does Ultima Template will receive update? Or I can upgrade to Angular 18 without waiting updates for Ultima?

**Comment by nazrulcsebd** [comment]
Have any updates on **Ultima** for Angular 18/19 and PRIMENG 18

**Reply by kashcode** [reply]
Need wait. I know there will be update for Ultima template.

**Comment by nazrulcsebd** [comment]
Do you have any updates on Ultima for Angular 18/19 and PRIMENG 19? The year,24 already ended. Please give us a timeline,  when the 19 version released?

**Comment by rene-schakmann** [comment]
same question here, waiting for the update
